instrumentation/ratpack/ratpack-1.7/library/src/main/java/io/opentelemetry/instrumentation/ratpack/v1_7/internal/RatpackHttpAttributesGetter.java

```diff
@@ -13,7 +13,11 @@
 import ratpack.http.Response;
 import ratpack.server.PublicAddress;
 
-enum RatpackHttpAttributesGetter implements HttpServerAttributesGetter<Request, Response> {
+/**
+ * This class is internal and is hence not for public use. Its APIs are unstable and can change at
+ * any time.
+ */
+public enum RatpackHttpAttributesGetter implements HttpServerAttributesGetter<Request, Response> {
```

Comment:
doesn't need to be public because the only class that uses it was also moved to the same package